ui/ui.go

```diff
@@ -0,0 +1,48 @@
+package ui
+
+import (
+	"fmt"
+
+	"github.com/manifoldco/torus-cli/prefs"
+)
+
+const (
+	InfoLevel  = 3
+	ErrorLevel = 1
+)
+
+// LogLevel represents the current allowed level of output
+var LogLevel int
+
+// EnableProgress is whether progress events should be displayed
+var EnableProgress = true
+
+// EnableHints is whether hints should be displayed
+var EnableHints = true
+
+func init() {
+	preferences, _ := prefs.NewPreferences(true)
```

Comment:
You're saying a custom init function?

Cause it would definitely suck to pass prefs every time you were doing an output.